Userland/Libraries/LibWeb/DOM/Document.cpp

```diff
@@ -628,6 +630,60 @@ NonnullRefPtr<Range> Document::create_range()
     return Range::create(*this);
 }
 
+// https://dom.spec.whatwg.org/#dom-document-createevent
+NonnullRefPtr<Event> Document::create_event(const String& interface)
```

Comment:
`interface` should be made all lowercase, as it needs to be a case-insensitive match.

Reply:
`matches()` defaults to `CaseSensitivity::CaseInsensitive`

Reply:
Oh, didn't know that :)
I just realised is_one_of is case sensitive so it's probably better to leave it all as matches then.

Reply:
now that I think about it, matches is probably the wrong method to use here as it's really intended for globs. `equals_ignoring_case()` would do the trick but just uses `to_lowercase` under the hood, so we'd call that a bunch - so we're back to your initial suggestion :smile:

Reply:
Now implemented using explicit `to_lowecase` and `is_one_of` where applicable. Thanks for the suggestions!